tests/Testdata/pos3.json

```
@@ -0,0 +1,11 @@
+{
+	"fullWidth": 21243,
```

Please indent the lines evenly.

```suggestion
  "fullWidth": 21243,
```

---

A bit unfair since VSCode shows them all aligned :smile: Reformatted ...